build-system/pr-check/utils.js

```diff
@@ -57,25 +59,28 @@ function printChangeSummary(fileName) {
   if (isTravisBuild()) {
     console.log(
       `${fileLogPrefix} ${colors.cyan('origin/master')} is currently at ` +
-        `commit ${colors.cyan(shortSha(gitTravisMasterBaseline()))}`
+        `commit ${colors.cyan(
+          shortSha(gitTravisMasterBaseline(), TRAVIS_SHA_LENGTH)
+        )}`
     );
     commitSha = travisPullRequestSha();
   } else {
     commitSha = gitCommitHash();
   }
   console.log(
     `${fileLogPrefix} Testing the following changes at commit ` +
-      `${colors.cyan(shortSha(commitSha))}`
+      `${colors.cyan(shortSha(commitSha, TRAVIS_SHA_LENGTH))}`
   );
 
   const filesChanged = gitDiffStatMaster();
   console.log(filesChanged);
 
-  const branchPoint = gitMergeBaseMaster();
+  const branchCreationPoint = gitBranchCreationPoint();
   console.log(
-    `${fileLogPrefix} Commit log since branch ` +
-      `${colors.cyan(gitBranchName())} was forked from ` +
-      `${colors.cyan('master')} at ${colors.cyan(shortSha(branchPoint))}:`
+    `${fileLogPrefix} Commit log since branch`,
+    `${colors.cyan(gitBranchName())} was forked from`,
+    `${colors.cyan('master')} at`,
+    `${colors.cyan(shortSha(branchCreationPoint, TRAVIS_SHA_LENGTH))}:`
```

Comment:
Can we keep the sha length here consistent with the git log's sha length? I had to do a double take because `e7c8dfbad` and `e7c8dfb` (from this PR's Travis logs) look different at first.

Reply:
Turns out `git` uses different lengths on different platforms. I was wrong. Reverted this change.

Reply:
@danielrozenberg @estherkim I was puzzled about this until I realized that `git` uses 7 or 9 characters in shortened SHAs depending on the length of the commit log.

7 chars for short commit logs: https://travis-ci.org/ampproject/amphtml/jobs/534993664#L323
9 chars for lengthy commit logs: https://travis-ci.org/ampproject/amphtml/jobs/534997714#L321

Oh well!